phpBB/includes/acp/acp_extensions.php

```diff
@@ -22,35 +22,37 @@
 class acp_extensions
 {
 	var $u_action;
+	var $tpl_name;
```

Comment:
Consistency: in other acp modules, the situation is:
- $u_action is mostly declared as var, with some instances as public (very few)
- $tpl_name is used but never declared
- $page_title is used but never declared
What should we do?  Keep consistency (do as above), or do it "right" (declare all three as public)?  Or any other alternative?